tests/FVTTests/homeView/testHomeView.ts

```diff
@@ -75,7 +68,7 @@ describe('JES explorer function verification tests', function () {
 
     before('Initialise', async () => {
         driver = await getDriver();
-        await setApimlAuthTokenCookie(driver, USERNAME, PASSWORD, `${BASE_URL}/api/v1/gateway/auth/login`, BASE_URL_WITH_PATH);
+        // await setApimlAuthTokenCookie(driver, USERNAME, PASSWORD, `${BASE_URL}/api/v1/gateway/auth/login`, BASE_URL_WITH_PATH);
```

Comment:
If the get driver is function is doing the setApimlAuthToken now we can remove this rather than commenting it out

Reply:
removed